examples/general/simple-web/README.md

```diff
@@ -2,3 +2,20 @@ An absolute bare-bones web app.
 
 Created from templates made available by Stagehand under a BSD-style
 [license](https://github.com/dart-lang/stagehand/blob/master/LICENSE).
+
+Run this example first to make sure all your tooling is working.
+
+## Detailed instructions (as of Nov2019) for Android Studio
+
+0. Clone the github repo for the `pwa` package.
+1. Install the Dart SDK (https://dart.dev/get-dart) (if you haven't already). This is different from the Flutter SDK.
+2. Run `brew info dart` in terminal and confirm no errors. Note the path to the Dart SDK. On mac OSX, this was `/usr/local/opt/dart/libexec`; Or, note where your Dart SDK is installed if you used another method.
```

Comment:
This is a Mac-specific instruction. Please make that clear or use more generic instruction.